### PR TITLE
FORMS-8828 items tab in edit dialog of wizard and vertical tabs

### PR DESCRIPTION
**Pull request by barshat7**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/.content.xml [hunk 1]
```diff
@@ -20,5 +20,5 @@
     jcr:description="Adaptive Form Vertical Tabs Container component, used to group other components and apply a common style or layout."
     jcr:primaryType="cq:Component"
     jcr:title="Adaptive Form Vertical Tabs (v1)"
-    sling:resourceSuperType="core/fd/components/form/panelcontainer/v1/panelcontainer"
+    sling:resourceSuperType="core/wcm/components/panelcontainer/v1/panelcontainer"
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/.content.xml:23]: As discussed in scrum, this is something which is not on deprecation line, right?

**Comment by rismehta** [comment]
@barshat7 Please check @sakshi-arora1 comments

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/.content.xml (hunk 1 above)
Comment (barshat7) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/.content.xml:23]: As of now none of the API's are listed as deprecated.

**Review by sakshi-arora1 — APPROVED** [review]

**Pull request merged**